lib/srv/desktop/ldap.go

```diff
@@ -113,6 +114,8 @@ func (c *ldapClient) create(dn string, class string, attrs map[string][]string)
 				return trace.BadParameter("object constraint violation on %q: %v", dn, err)
 			case ldap.LDAPResultInsufficientAccessRights:
 				return trace.AccessDenied("insufficient permissions to create %q: %v", dn, err)
+			case ldap.ErrorNetwork:
+				return trace.ConnectionProblem(err, "LDAP network error")
```

Comment:
Looks like all other `trace.ConnectionProblem` that were added contain the `dn`, should we add it here as well?

Reply:
Will do.